### DISCUSSION TITLE
Wanted | 招聘帖（社招、校招、实习）

### DISCUSSION THREAD

**Discussion by yanglbme** [doocs/advanced-java#156] [Show and tell] [answered]
> 候选人投递简历前，请务必确认招聘人信息，防止个人信息泄露给陌生人。招聘人请统一按以下格式创建招聘信息，谢谢。

## 公司/团队介绍
...

## 联系方式
- 邮箱
- 微信/手机号
- 其它

注：最好提供企业邮箱，证明身份。

**Comment by yanglbme** [comment]
## 公司/团队介绍
我们是 AliExpress 用户增长团队，不知道 AliExpress？你可以认为是对面歪果仁的淘宝，用户增长大有可为，想了解更多的话，直接联系我吧，我给你一对一的介绍。

注：本信息由 @ding102992 提供，见 #155。

## 联系方式
- 邮箱：mingmeng.dmm@alibaba-inc.com
- 微信：ding102992

**Comment by chenqimiao** [comment, marked as answer]
## 公司/团队介绍
天猫国际是中国消费升级的第一跨境平台，是阿里经济体5年2000亿美金进口承诺的主力军。2019年天猫国际技术部和考拉合并成立了阿里巴巴大进口技术部，是阿里巴巴国际化战略的核心技术部门。致力于进口业务的技术突破和创新，助力中国的消费者实现“买遍全球”的需求，跨入未来的万亿级市场。想了解更多的话，直接联系我吧，我帮你组内直推，大量HC，走过路过不可错过。
注：本信息由 @hunterzhang86 提供，见 #157。

## 联系方式
- 邮箱：yifeng.zh@alibaba-inc.com
- 微信：isHunterZhang

**Comment by Nerom** [comment]
阿里直招，社招校招均有，BU囊括电商零售系统全链路，前端至交易、详情页，后端至仓储、履约，更有海外复杂场景等你实践，有意请邮件简历至 liuzhu.wlz#alibaba-inc.com
校招的潜力股，勇敢发来简历吧，会邀请进校招小群，定期分享写简历的技巧，更有优先面试的机会
# 公司/团队介绍
我们是阿里巴巴集团新零售技术旗下供应链技术国际化团队，我们致力于通过世界一流的全球化技术服务来自全球10亿+的海外消费者，为全球消费者带去极致的购物体验，让买全球卖全球的使命落地成为普惠全球的显示图景。我们服务的业务实体有 lazada、AliExpress、天猫海外、Daraz、Trendyol等平台。
# 联系方式
邮箱： liuzhu.wlz@alibaba-inc.com

**Comment by ZhenghaeHo** [comment]
> ## 公司/团队介绍
> 我们是 AliExpress 用户增长团队，不知道 AliExpress？你可以认为是对面歪果仁的淘宝，用户增长大有可为，想了解更多的话，直接联系我吧，我给你一对一的介绍。
> 
> 注：本信息由 @ding102992 提供，见 #155。
> 
> ## 联系方式
> * 邮箱：[mingmeng.dmm@alibaba-inc.com](mailto:mingmeng.dmm@alibaba-inc.com)
> * 微信：ding102992

base在哪个城市

**Reply by ding102992** [reply]
杭州，西溪园区

**Comment by ZhenghaeHo** [comment]
> 阿里直招，社招校招均有，BU囊括电商零售系统全链路，前端至交易、详情页，后端至仓储、履约，更有海外复杂场景等你实践，有意请邮件简历至 liuzhu.wlz#alibaba-inc.com
> 校招的潜力股，勇敢发来简历吧，会邀请进校招小群，定期分享写简历的技巧，更有优先面试的机会
> 
> # 公司/团队介绍
> 我们是阿里巴巴集团新零售技术旗下供应链技术国际化团队，我们致力于通过世界一流的全球化技术服务来自全球10亿+的海外消费者，为全球消费者带去极致的购物体验，让买全球卖全球的使命落地成为普惠全球的显示图景。我们服务的业务实体有 lazada、AliExpress、天猫海外、Daraz、Trendyol等平台。
> 
> # 联系方式
> 邮箱： [liuzhu.wlz@alibaba-inc.com](mailto:liuzhu.wlz@alibaba-inc.com)

@Nerom base在哪个城市

**Comment by ZhenghaeHo** [comment]
@

> ## 公司/团队介绍
> 天猫国际是中国消费升级的第一跨境平台，是阿里经济体5年2000亿美金进口承诺的主力军。2019年天猫国际技术部和考拉合并成立了阿里巴巴大进口技术部，是阿里巴巴国际化战略的核心技术部门。致力于进口业务的技术突破和创新，助力中国的消费者实现“买遍全球”的需求，跨入未来的万亿级市场。想了解更多的话，直接联系我吧，我帮你组内直推，大量HC，走过路过不可错过。
> 注：本信息由 @hunterzhang86 提供，见 #157。
> 
> ## 联系方式
> * 邮箱：[yifeng.zh@alibaba-inc.com](mailto:yifeng.zh@alibaba-inc.com)
> * 微信：isHunterZhang

@chenqimiao base在哪

**Comment by He-Pin** [comment]
## 公司/团队介绍
阿里巴巴消息业务/基础团队招聘（校招/社招）直招，社招校招均有。
商业链接，通过消息，场景触达、链接商家和消费者。为商家的经营决策做贡献。
在提供手淘、天猫的消息、营销推送的能力同时；同时，我们也提供了开放的二方支持能力，支持直播、钉钉直播、钉钉海外直播、手淘、天猫、闲鱼、飞猪、零售通、躺平等二方App的消息能力。
招聘岗位：Java 后端、Android客户端、iOS客户端、CPP跨终端，见[详情](https://github.com/CampusCenter/JobBoard/issues/1)

## 联系方式
微信：kerisle
邮箱： hepin.p@alibaba-inc.com
base: 杭州、阿里巴巴西溪园区

**Comment by yanglbme** [comment]
### **公司/团队介绍**
我们是涂鸦智能工程生产力团队，无论你是java、python、测试、安卓、前端还是go，我们都招，具体岗位详情如下
坐标：杭州西湖区华策中心A座 

**一. 高级Java开发工程师**

岗位职责：
1、负责IoT测试基础设施建设，主要包括云端链路测试及稳定性监控平台、性能测试
平台、嵌⼊式固件平台；
2、优化各平台定时任务调度、任务执⾏器、⽇志采集系统，保证⾼并发稳定性、可
⽤性、实时性；
3、深⼊理解公司云端、嵌⼊式、测试相关业务流程，分析问题，制定解决⽅案，进
⾏架构设计并⾼效实施满地；
4、对技术有热情，持续学习，乐于熟悉新的技术产品。 
岗位要求：
1、熟悉Java语言，对相关技术领域的开源产品有深入的理解；
2、熟悉分布式、性能调优、缓存、消息、搜索等机制；
3、熟悉MySQL等数据库中的一种或多种，有数据库调优经验，了解MySQL及相关分布式存储技术；
4、思维清晰，头脑灵活，能独立分析并解决复杂的线上问题；
5、三年以上Java相关的开发经验。
加分项：
1）熟悉Golang者优先；
2）熟悉python/ruby等脚本语言者优先；
3）有K8S相关经验者优先；



**二. Android开发工程师**

岗位描述

1）独立完成 Android客户端程序的开发，相关功能预研和研究；

2）根据产品需求设计并开发相关的移动产品；

3）验证和修正测试中发现的问题；

4）一些终端命令工具开发;

岗位要求：

1）计算机或相关专业本科以上学历；

2）对数据结构和算法设计具有深刻的理解；

3）熟悉掌握Java，熟悉Android SDK；

4）一年以上Android开发经验，能独立开发Android App；

5）善于学习和运用新知识，具有良好的分析和解决问题能力；

6）具有良好的团队合作精神和积极主动的沟通意识；

加分项：

1）熟悉Appium；

2）有Flutter开发经验；

3）熟练运用算法和数据结构；

4）熟悉Python/Ruby等脚本语言；

5）有Golang经验，终端工具开发经验者优先

6）有性能优化项目经验。



**三. Golang开发工程师**

职位描述：

1、负责设计, 完善内部CI系统, 包管理工具研发等，结合需求设计实现安全、稳定、高伸缩性、高性能、易维护、易用性好的业务系统

2、根据系统发展，对技术架构不断调整，持续优化。

3、与运维同学合作，实现平台服务的可用性管理，如变更、故障、监控、容量等方向的自动化与平台化工作

职位要求：

1 、熟悉算法、数据结构和常用的设计模式；

2、2-5年开发工作经验, 具有良好的英文阅读能力, 熟悉Golang生产环境开发工作经验, 熟悉 Golang 并发编程

3、熟悉分布式数据库的设计原理，熟悉NoSQL，有 RocksDB、Redis、MongoDB等常见NoSQL数据库的实际使用和开发经验

4、熟悉Docker基本原理及工具使用, 熟悉K8S 者加分

5、熟悉Socket 网络编程者优先

6、具有良好的技术钻研能力，能够深入底层解决技术难题

7、有强烈的责任心和团队精神，善于沟通和合作



**四. 测试开发工程师**

岗位职责：

1、参与云服务接口、物联网设备接口测试；

2、参与或构建与业务匹配的自动化框架及自动化解决方案，引领团队建立物联网的自动化测试体系；

3、参与或构建云服务的持续集成，与开发、运维一起构建公司的CI、CD流程和系统；

4、基于在线工具平台建立各种小工具，提高测试效率；

5、可参与到业务中提供测试架构、流程、工具等方面的技术支持。

岗位要求：

1、具备较强的分析、解决问题的能力，熟练掌握至少一种常用编程语言，如Java、Python、C++、C#等；

2、对常用协议如HTTP、Dubbo等有一定了解，熟练掌握多种测试工具，如JMeter、Charles、Jira、禅道等；

3、对测试感兴趣，很强的责任心，良好的沟通能力和表达能力，能承受一定的工作压力；

4、有过1年以上大型服务测试经验者，或3年以上测试开发经验者优先；

5、有使用过常用测试框架JUnit、TestNG、PYUnit、JMeter性能扩展等并对框架有定制能力者优先；

6、有物联网自动化测试经验、Golang、或对QTPP协议有了解或涂鸦生态开发者优先。



**五. Python开发工程师**

职位描述： 

1、负责功能设计、开发、维护 

2、根据系统发展，对技术架构不断调整，持续优化

职位要求：

 1 、至少一年以上python web端后台开发经验

 2、熟练掌握Django、[Web.py](http://web.py/)、Tornado、Flask等至少1个Web开发框架 

3、有丰富web后端系统性能调优经验优先 

4、熟悉Linux环境下的开发 

5、熟悉mysql数据库，有数据库调优经验的优先 

6、具有良好的技术钻研能力，能够深入底层解决技术难题 7、有强烈的责任心和团队精神，善于沟通和合作




**六. 前端开发工程师** 

岗位描述：

我们负责涂鸦IoT平台开发，目标是通过前端技术，让物联网行业的数据、应用可以高效、自由的开发和流转。；

我们关注： 

1. Web应用、组件库、数据可视化、工程工具、BFF等；

2. 网站、平台、无线等业务的前端开发； 

3. 设计并实现物联网场景下，面向开发者的应用开发框架；

工作内容：

1. 参与PC、APP Hybrid、小程序、数据可视化等前端开发；

2. 参与改进开发、构建、发布等前端工程化体系；

3. 参与公司内部系统建设（包括Node全栈开发）；

任职要求：

1. 本科及以上学历，有至少1年以上前端工作经验；

2. 熟练使用HTML、CSS、JavaScript等前端技术，熟悉HTTP协议以及浏览器原理；

3. 至少熟练运用一种前端开发框架并掌握其原理如: Vue.js/React.js及其生态圈；

4. 具备良好的沟通能力和优秀的团队协作精神；

符合以下任一条件者优先：

熟悉一门非前端的语言 如:Java/PHP/Python/Ruby等，并有实践经验;

参与开源项目并贡献过代码，附上Github链接；

熟练使用 git、github、linux 等工具。

### **联系方式**
shuxiaotai@gmail.com

注：本信息由 @shuxiaotai  提供，见 #202。

**Comment by LucyDoggy** [comment]
**公司：国内一线量化交易私募，百亿私募，产品平均年收益30~50%**
地点：上海/北京/深圳/香港（各岗位地点不同）

**量化研究员＜算法＞**
Comments
1. 知名高校CS/数学/物理专业，精通深度学习/时间序列/神经网络及一些比较前沿的算法技术；应用方向不限，推荐搜索图像语音语义数据挖掘工业算法自动驾驶等等不限制行业，只看技术。
2. 可以没有量化经验，有互联网行业算法经验对金融有兴趣；薪资对标阿里P8P7，但轻松不加班，优秀应届也欢迎； 总之，薪资具体得看个人情况。
如果有量化经验成熟策略，薪资另谈。
3. 上海为主，北京香港杭州少量HC
＜Time series/predictive model/reinforcement Learning/meta learning/deep learning＞

工作职责
1、 利用机器学习/深度学习/强化学习方法进行量化投资策略研发；
2、 紧跟相关领域前沿，推动基础研究；
3、 通过设计开拓性的新的深度神经网络等方式改进已有的机器学习算法，对二级市场标的进行特征/价格预测；
4、 构建科学、严谨的算法评测体系；
5、协助相关交易策略程序的编写、维护、调试、优化。

任职资格
1、海内外知名高校硕士或博士学历，计算机、数学、统计等机器学习相关专业，有扎实的理论功底，掌握机器学习领域常用算法，曾发表高水平学术论文者优先；
2、在机器学习/深度学习/强化学习方面有一年以上的实战经验，例如在语音、图像识别等方面有过工作或者研究经验，有量化研究经历者优先（非必须）；
3、熟练掌握Python，以及相关的开发工具包，如tensorflow、kears；
4、熟练使用Hadoop、Hive、SQL、Spark进行数据处理和分析；
5、有较强的学习能力、自我驱动力、业务理解和解决问题的能力；

**Golang_服务端资深开发_杭州_量化交易**
【职责描述】
（1）构建和优化公司分布式高性能计算、虚拟化、网络、存储等基础设施
（2）保证基础设施的可用性和稳定性
【岗位要求】
（1）具备分布式系统架构设计开发能力，掌握容器化、消息中间件等核心技术
（2）熟悉日常使用工具的原理及实现，能独立设计并完成模块开发和交付工作
（3）具备扎实的编程能力、优秀的设计能力和代码品味，具有强烈的责任心
（4）精通golang，熟练掌握python，有k8s开发经验者优先

### ###急！ **C++后端/c++底层开发_招聘_上海北京深圳香港**

薪资：40-160W+<视经验及个人情况>；
要求：c++后端 熟悉低延时高性能交易系统开发 /互联网大数据分布式 /游戏等，关注 C++前沿技术（比如，了解 C++ 17/21); 优秀应届生也可； c++底层<特别视个人情况> 有无锁编程 /视频编解码 /精通汇编语言等性能调优能力，尤其 HPC，以及熟悉硬件 FPGA 等原理；
公司：高频量化私募（机器学习为主）；

**联系人**：Lucy chen M:13701809157 WECHAT:13701809157 MAIL:luciachenbingqing@outlook.com 由于不是经常在线，直接打我电话或者微信我就好了

**Comment by edricgsh** [comment]
# 公司/团队介绍

蚂蚁国际团队主要负责国外地区的支付产品建设。复用蚂蚁在国内成熟的支付底层能力，为不同国家的支付产品赋能，拿到战略性的成果。

现在在新加坡开启了校招和社招，目前有大量HC，欢迎有兴趣的小伙伴可以把简历投过来：siehuai.gan@antgroup.com

招聘岗位：Java 后端, 后端测试工程师

## 职位诱惑
1、薪资高，丰厚现金收入，全球超低税率
2、舞台大，足够你为之奋斗一辈子也无法触碰到公司的天花板。
3、平台好，在这里呆上三四年，对个人能力的培养将不亚于读完一所一流的大学，甚至会超过。
4、结识牛人，在这里你会遇到大量极其优秀的人才、牛人。
5、长见识，你会和来自各个国家的同学一起奋斗，你也可以出差到全世界各地，边旅游边工作。

## 职位描述
1、负责支付宝国际支付钱包的相关开发工作；
2、参与需求的讨论、设计、开发；
3、配合各个部门间的各种讨论、评审、测试、运维等；
4、负责一到两个现有系统的运维、升级改造。

## 职位要求
1、扎实的java编程基础，熟悉分布式应用开发及相关开源技术；熟悉linux及常用命令，以及常用数据库；
2、2年以上大规模高并发访问的Web应用系统设计、开发工；
3、有较强的表达和沟通能力，有一定的领导力和创新意识，责任心强、抗压能力强，对质量要求苛刻，对技术的追求不仅仅停留在使用

## Base 地
新加坡

## 联系方式
邮箱：siehuai.gan@antgroup.com
